01_getting_started/hello_world.py

```diff
@@ -49,13 +46,13 @@ def f(i):
 #
 # Now let's see three different ways we can call that function:
 #
-# 1. As a regular `local` call on your computer, with `f.local`
+# 1. As a regular `local` call on your computer, with `f.local()`
```

Comment:
Happy to accept this change, but want to note that to me, `f.local()` reads as "the _output_ of invoking the function `f.local()`", not as "the function `f.local` itself".

I've heard both opinions from others, but outside of a functional programing context, the use of parens to denote that the object being discussed is a function is more popular.